src/lib/sepana/api/generic.ts

```diff
@@ -5,23 +5,59 @@ import { CURRENT_VERSION } from '../constants'
 import { SEPANA_ENDPOINTS } from './endpoints'
 import { sepanaAxios } from './http'
 
+const MAX_SEPANA_PAGE_SIZE = 100
+
 /**
  * Exhaustively queries all Sepana records.
  *
  * @returns Promise containing all records from Sepana database
  */
 export async function queryAll<T extends object>() {
```

Comment:
sepana API now only supports a max page size of 100, we have to use some paging logic here instead of our previous pageSize of 10000

details: this is complicated bc for some reason sepana API doesn't use a pointer when returning hits for a page number... meaning if you pass a pageSize of 100, you'll _always_ get 100 hits back even if you're at the end of the records or past it. if/when they _start_ using a pointer, this will still work fine